presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -370,4 +373,22 @@ public void testCorrelatedScalarAggregationRewriteToLeftOuterJoin()
                                                                                 project(ImmutableMap.of("NON_NULL", expression("true")),
                                                                                         node(ValuesNode.class)))))))))));
     }
+
+    @Test
+    public void testCountStarOverInnerAggregateQuery()
+    {
+        assertPlan(
+                "SELECT count(*) from (select sum(orderkey) from orders)",
+                output(
+                        values(ImmutableList.of("_col0"), ImmutableList.of(ImmutableList.of(new LongLiteral("1"))))));
+    }
+
+    @Test
+    public void testCountColumnOverInnerAggregateQuery()
```

Comment:
I would merge this two methods together and mention which rule you are testing in the method name